bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -23,9 +23,6 @@ If your system language is supported by the OpenWeatherMap API it will be used a
 The second thing `weather-and-forecast` supports the [current weather](https://openweathermap.org/current), [5 day / 3 hour forecast](https://openweathermap.org/forecast5) and optional [16 day / daily forecast](https://openweathermap.org/forecast16) services for a specific location.
 It requires coordinates of the location of your interest.
 You can add as many `weather-and-forecast` things for different locations to your setup as you like to observe.
-**Attention**: The daily forecast is only available for [paid accounts](https://openweathermap.org/price).
```

Comment:
I am not sure if this is correct. The One Call API supports daily forecast for free, but this section is explaining the legacy Thing querying a different API endpoint which is still limited. The Thing for the One Call API is described in the section below.

Reply:
According to https://openweathermap.org/price there is no difference for daily forecasts in free and paid accounts.

Reply:
I disagree. From my pov "Free" plan includes Daily forecast 7 days provided by the One Call API (v3.0). All higher plans additionally provide Daily forecast 16 days provided by the v2.5 version of the API.